eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -31,24 +31,24 @@ jobs:
           BuildSHA: $(Build.SourceVersion)
           RepoId: 'Azure/azure-sdk-for-net'
       - script: >-
-          dotnet pack eng/service.proj -o $(Build.ArtifactStagingDirectory) -warnaserror /p:ServiceDirectory=${{parameters.ServiceToBuild}}
```

Comment:
It is probably about time we split this to our newer standard of one argument per line.